docs/_posts/2016-07-08-introducing-reacts-error-code-system.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Introducing React's Error Code System"
+author: keyanzhang
+---
+
+Building a better developer experience has been one of the things that React deeply cares about, and a crucial part of it is to detect anti-patterns/potential errors early and provide helpful error messages when things (may) go wrong. However, most of these only exist in development mode; in production, we avoid having extra expensive assertions and sending down full error messages in order to reduce the number of bytes sent over the wire. Prior to this release, we stripped out error messages at build-time and this is why you see the "Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings." message in production.
```

Comment:
Can we break it out into a blockquote to aid skimming?

---

this is why you might have seen this message in production:

> Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.